v22.1/demo-fault-tolerance-and-recovery.md

```diff
@@ -379,33 +361,18 @@ kill -TERM 4622
     $ pkill haproxy
     ~~~
 
-3. Gracefully shut down the remaining 4 nodes, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):
+3. Gracefully shut down the remaining **4 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure) and the new process ID for the node stored in `fault-node5`:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    kill -TERM 4482
-    ~~~
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    kill -TERM 4497
+    kill -TERM {process-id}
```

Comment:
```suggestion
    kill -TERM 53645 53651 53671 53684
```

Reply:
Same comment as above. The instructions say to shut down the remaining 4 nodes and I give the command for that here. The users\'s process IDs will be different.

Reply:
Same suggestion as above:

{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM {process IDs}
~~~